### PR TITLE
set a bunch of keywords requested by DM

### PR DESCRIPTION
**Pull request by jchiang87**

### CODE REVIEW

File: python/lsst/sims/GalSimInterface/galSimDetector.py [hunk 1]
```diff
@@ -90,6 +101,28 @@ def __init__(self, detectorName, cameraWrapper, obs_metadata, epoch, photParams=
         if self.obs_metadata.rotSkyPos is not None:
             self.fitsHeader.set('ROTANGLE', obs_metadata.rotSkyPos)
 
+        # Add airmass, needed by jointcal.
+        try:
+            airmass = self.obs_metadata.OpsimMetaData['airmass']
+        except KeyError:
+            pass
+        else:
+            self.fitsHeader.set('AIRMASS', airmass)
```
Comment (cwwalter) [inline python/lsst/sims/GalSimInterface/galSimDetector.py]: Josh had requested 

```
• parallactic angle
• temperature
• pressure
• water-vapor partial pressure
to the list if they're available, as these all also affect DCR.
```

Are they available from the same OpSim information at this point?

File: python/lsst/sims/GalSimInterface/galSimInterpreter.py [hunk 2]
```diff
@@ -670,8 +665,7 @@ def getHourAngle(self, mjd, ra):
     @property
     def observatory(self):
         if self._observatory is None:
-            self._observatory \
-                = LsstSimMapper().MakeRawVisitInfoClass().observatory
+            self._observatory = LsstObservatory()
```
Comment (cwwalter) [inline python/lsst/sims/GalSimInterface/galSimInterpreter.py]: I couldn't figure out if LSSTObservatory was a singleton or not (as it is being used in more than one file).  Is that handled through LsstSimMapper via LsstSimMapper().MakeRawVisitInfoClass().observatory?

Also, it looks like this is a place we still need obs_lsstSim is that right?

File: python/lsst/sims/GalSimInterface/galSimDetector.py (hunk 1 above)
Comment (jchiang87) [reply python/lsst/sims/GalSimInterface/galSimDetector.py]: None of these are available.

File: python/lsst/sims/GalSimInterface/galSimInterpreter.py (hunk 2 above)
Comment (jchiang87) [reply python/lsst/sims/GalSimInterface/galSimInterpreter.py]: > it looks like this is a place we still need obs_lsstSim is that right?
yes.

File: python/lsst/sims/GalSimInterface/galSimDetector.py [hunk 3]
```diff
@@ -73,14 +78,20 @@ def __init__(self, detectorName, cameraWrapper, obs_metadata, epoch, photParams=
         if self.obs_metadata.bandpass is not None:
             if (not isinstance(self.obs_metadata.bandpass, list) and not
                 isinstance(self.obs_metadata.bandpass, np.ndarray)):
-
                 self.fitsHeader.set("FILTER", self.obs_metadata.bandpass)
 
         if self.obs_metadata.mjd is not None:
             self.fitsHeader.set("MJD-OBS", self.obs_metadata.mjd.TAI)
+            mjd_obs = astropy.time.Time(self.obs_metadata.mjd.TAI, format='mjd')
```
Comment (timj) [inline python/lsst/sims/GalSimInterface/galSimDetector.py]: If you are using TAI, do you set TIMESYS header somewhere?

File: python/lsst/sims/GalSimInterface/galSimDetector.py [hunk 4]
```diff
@@ -90,6 +101,28 @@ def __init__(self, detectorName, cameraWrapper, obs_metadata, epoch, photParams=
...
+        try:
+            airmass = self.obs_metadata.OpsimMetaData['airmass']
+        except KeyError:
+            pass
+        else:
+            self.fitsHeader.set('AIRMASS', airmass)
+
+        # Add boilerplate keywords requested by DM.
+        self.fitsHeader.set('TELESCOP', 'LSST')
+        self.fitsHeader.set('INSTRUME', 'CAMERA')
```
Comment (timj) [inline python/lsst/sims/GalSimInterface/galSimDetector.py]: I do wonder what we are going to really call our camera...

**Pull request merged**